moveit_ros/planning/rdf_loader/CMakeLists.txt

```
@@ -32,7 +32,8 @@ if(BUILD_TESTING)
 
   install(
     TARGETS
-      test_rdf_integration boring_string_publisher
+      test_rdf_integration
+      boring_string_publisher
```

I have a preference for not modifying files if you can help it. 
```suggestion
      test_rdf_integration boring_string_publisher
```